Sources/ApolloWebSocket/DefaultImplementation/WebSocket.swift

```diff
@@ -68,8 +68,11 @@ public final class WebSocket: NSObject, WebSocketClient, StreamDelegate, WebSock
     public let code: Int
   }
 
+  /// The WebSocket sub-protocols supported.
   public enum WSProtocol: CustomStringConvertible {
```

Comment:
I think these comments are interchanged for both protocols.

-   The transport protocol 'graphql-transport-ws' is implemented as per [graphql-ws](https://github.com/enisdenjo/graphql-ws) package.
-     The transport protocol 'graphql_ws' is implemented as per [subscriptions-transport-ws](https://github.com/apollographql/subscriptions-transport-ws) package.  This is considered a legacy implementation.

Reply:
Thanks @phanikavi, you're correct. Honestly the naming of these libraries and sub-protocols is a mess and an endless source of confusion. I'm going to rename the cases in `WSProtocol` to match [apollo-kotlin](https://github.com/apollographql/apollo-kotlin/tree/main/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/ws) and be closer to the library names instead. It's what most people will be familiar with rather than the sub-protocol names.